resources/js/components/base/Header.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div>
+        <slot></slot>
+    </div>
+</template>
+
+<script>
+export default {
+
+    computed: {
+        stickyNavbar: function () {
+            return document.querySelector('#stickyNavbar');
+        },
+
+        navOffsetTop: function () {
+            return this.stickyNavbar.offsetTop;
+        },
```

Comment:
Seria melhor adicionar um ref no elemento e acessar ele com this.$refs, assim não precisa de um computed para isso

```suggestion
        navOffsetTop () {
           return this.$refs.stickyNavbar.offsetTop;
        },
```

https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements

Reply:
mas a navbar tá definida no `<slot></slot>`, aí tem como usar refs no que está definido no slot?

Reply:
Real, não percebi, não sei exatamente como lidar com isso nesse caso.